src/ensembl/src/content/app/entity-viewer/types/slice.ts

```diff
@@ -0,0 +1,22 @@
+import { Strand } from './strand';
+
+export type Slice = {
+  location: {
+    start: number;
+    end: number;
+    // length: number,
```

Comment:
Can this be removed?

Reply:
It could... At the moment, I think, Kieron’s implementation has the `length` property in the `location` object; but I feel `length` is a property of the feature itself, not of location. Eventually, after further discussions, this property will have to end up either in the feature itself or in its location field; but in the meantime we can of course delete it.

Reply:
removed the comment